lib/ansible/modules/cloud/ovirt/ovirt_datacenter.py

```diff
@@ -188,6 +197,7 @@ def main():
         quota_mode=dict(choices=['disabled', 'audit', 'enabled']),
         comment=dict(default=None),
         mac_pool=dict(default=None),
+        force=dict(default=False, type='bool'),
```

Comment:
Default `false` is the default of the engine as well. So we can use here `None`, and don't send additional data via network, if it is not needed.